json-schema/schema.json

```diff
@@ -490,46 +490,40 @@
     "mlm:input": {
       "type": "array",
       "items": {
-        "title": "Model Input Object",
-        "type": "object",
-        "required": [
-          "name",
-          "bands",
-          "input"
-        ],
-        "properties": {
-          "name": {
-            "type": "string",
-            "minLength": 1
-          },
-          "bands": {
-            "$ref": "#/$defs/ModelBands"
-          },
-          "input": {
-            "$ref": "#/$defs/InputStructure"
-          },
-          "description": {
-            "type": "string",
-            "minLength": 1
-          },
-          "norm_by_channel": {
-            "type": "boolean"
-          },
-          "norm_type": {
-            "$ref": "#/$defs/NormalizeType"
-          },
-          "norm_clip": {
-            "$ref": "#/$defs/NormalizeClip"
-          },
-          "resize_type": {
-            "$ref": "#/$defs/ResizeType"
-          },
-          "statistics": {
-            "$ref": "#/$defs/InputStatistics"
-          },
-          "pre_processing_function": {
-            "$ref": "#/$defs/ProcessingExpression"
-          }
+        "$ref": "#/$defs/ModelInput"
+      }
+    },
+    "ModelInput": {
+      "title": "Model Input Object",
+      "type": "object",
+      "required": [
+        "name",
+        "bands",
+        "input"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 1
+        },
+        "bands": {
+          "$ref": "#/$defs/ModelBands"
+        },
+        "input": {
+          "$ref": "#/$defs/InputStructure"
+        },
+        "description": {
+          "type": "string",
+          "minLength": 1
+        },
+        "scaling": {
```

Comment:
I'm having some lingering concern that "scaling" will be confused with spatial scale. what do you think of renaming this to "value_scaling"?

Reply:
yes, we can do that

Reply:
ok awesome. good to merge after rename.

Reply:
Changed. Please review again just to make sure I didn't mess up any search/replace.